packages/boot/test/bootstrapTests/supports.js

```diff
@@ -324,6 +333,27 @@ export const makeProposalExtractor = ({ childProcess, fs }) => {
 };
 harden(makeProposalExtractor);
 
+export const matchRef = (t, ref1, ref2, message) =>
+  t.is(krefOf(ref1), krefOf(ref2), message);
```

Comment:
would be nice to not have to pass in `t`. Is the log output sufficient this way?
```suggestion
export const matchRef = (ref1, ref2, message) =>
  assert(krefOf(ref1) === krefOf(ref2), message);
```

Reply:
`t` is the Ava test object, which you need to run the test predicates else the test failures won't show up in the test report.

Reply:
Exception show up, but there's not pretty.
```js
import test from 'ava';

test('t is', t => {
  t.is(1, 2, 'my message');
});

test('assert.equals', t => {
  assert.equal(1, 2, 'my message');
});
```

```
$ ava test/test-utils.js

  ✘ [fail]: t is my message
  ✘ [fail]: assert.equals Error thrown in test
  ─

  t is

  my message

  Difference (- actual, + expected):

  - 1
  + 2

  › packages/internal/test/test-utils.js:5:5



  assert.equals

  Error thrown in test:

  RangeError {
    message: 'my message',
  }

  › packages/internal/test/test-utils.js:9:10

  ─

  2 tests failed
```

I agree this here is better. 

Though I look forward to https://github.com/avajs/ava/issues/1094